activities/MindMath.activity/js/activity.js

```diff
@@ -7,14 +7,26 @@ requirejs.config({
 });
 
 // Vue main app
-var app = new Vue({
-  el: '#app',
+const app = Vue.createApp({
+  // el: '#app',
   components: {
     'game': Game,
     'result': Result,
     'leaderboard': Leaderboard,
+    'slots': Slots,
+    'clock': Clock,
+    'sugar-activity': SugarActivity,
+    'input-number': InputNumber,
+    'sugar-toolitem': SugarToolitem,
+    'sugar-toolbar': SugarToolbar,
+    'sugar-journal': SugarJournal,
+    'sugar-localization': SugarLocalization,
+    'sugar-presence': SugarPresence,
+    'sugar-tutorial': SugarTutorial,
+    'sugar-popup': SugarPopup,
+    'sugar-icon': SugarIcon
   },
-  data: function() {
+  data() {
```

Comment:
Are you sure this syntax should be changed?
I guess `data: function() {` means the same.